src/components/ProductsList.js

```diff
@@ -1,6 +1,14 @@
-const ProductsList = () => {
+import Product from "./Product";
+
+const ProductsList = ({products, addToCart}) => {
   return (
-    <h1>Products List</h1>
+    <>
+      {products.map(product =>
+        <div key= {product.name}>
```

Comment:
Acho que rolou um typo aqui:

```suggestion
        <div key={product.name}>
```